app/models/motion.rb

```diff
@@ -140,7 +140,7 @@ def members_count
     if voting?
       group.memberships_count
     else
-      self[:members_count]
+      self[:members_count] || 0
```

Comment:
I suggested this approach but now I think the db schema should be changed to default: 0 with null: false

Hannah if you don't mind, can you try giving that a go?